prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -27,6 +29,153 @@ def check_for_location_conflicts(substations):
         )
 
 
+def map_lines_to_substations_using_coords(substations, lines, **kwargs):
+    """Map lines to substations using coordinates
+
+    :param pandas.DataFrame substations: data frame of substations.
+    :param pandas.DataFrame lines: data frame of all lines.
+    :param int \\*\\*kwargs: optional arguments.
```

Comment:
Just to confirm, according to the discussion below, we are going to use a user facing dictionary as a collections of all `kwargs`, then here we just look for `rounding` right? Then I presume this is a `dict` instead of `int`, no?

Reply:
not sure how the `int` sneaked in

Reply:
Probably originally `:param int rounding:`

Reply:
Used `:param **kwargs:`